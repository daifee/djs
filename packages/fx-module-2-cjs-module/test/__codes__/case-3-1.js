

fxRequire.async('case-1-3', (mod) => {
  console.log(mod);
}, (err) => {
  console.log(err);
});