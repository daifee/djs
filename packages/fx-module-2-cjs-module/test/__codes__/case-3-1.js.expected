"use strict";

import('case-1-3').then(mod => {
  console.log(mod);
}).catch(err => {
  console.log(err);
});